gradle.properties

```diff
@@ -1,5 +1,5 @@
 version=7.1.1
-iexecCommonVersion=6.0.0
+iexecCommonVersion=6.0.0-NEXT-SNAPSHOT
```

Comment:
Why do we need next?

Reply:
I don't know, it was already done when I inherited from the branch.
I reverted this change in 70ff3c3.